docs/faqs/max_execution_time.rst

```diff
@@ -1,74 +1,85 @@
 .. _faqs/max_execution_time:
 
-============================================================
-What is the maximum execution time for a Qiskit Runtime job?
-============================================================
+=======================================================================
+What is the maximum execution time for a Qiskit Runtime job or session?
+=======================================================================
 
-To ensure fairness, there is a maximum execution time for each Qiskit Runtime job. If
-a job exceeds this time limit, it is forcibly cancelled. This is represented in the job
-status as `Cancelled - Ran too long`. The maximum execution time is the
-smaller of these values: 
+Job maximum execution time
+***************************
 
-- The system limit 
-- The ``max_execution_time`` defined by the program
-- The ``max_execution_time`` defined by the job itself
+To ensure fairness, and as a way to help control cost, there is a
+maximum execution time for each Qiskit Runtime job. If
+a job exceeds this time limit, it is forcibly cancelled and a ``RuntimeJobMaxTimeoutError``
+exception is raised.
 
-The system limit depends on the channel you're using and is defined as follows:
+.. note::
+   As of ``qiskit-ibm-runtime`` 0.12.0, the ``max_execution_time`` value is based on quantum
+   time instead of wall clock time. Quantum time represents the time that the QPU
+   complex (including control software, control electronics, QPU, and so on) is engaged in
+   processing the job.
 
-IBM Cloud system limit
-***************************
+   Simulator jobs continue to use wall clock time because they do not have quantum time.
 
-The system limit on the Qiskit Runtime job execution time is 3 hours for a job running on a simulator
-and 8 hours for a job running on a physical system.
```

Comment:
By removing this there is no mention in these docs about the cloud channel for pay-go and limits imposed there. Is that intentional? Because otherwise this doc reads as IQP-only to me, i.e. "Premium User" is not a pay-go cloud channel concept.